roles/infra/tasks/main.yml

```
@@ -21,7 +21,7 @@
     state: latest
```

Comment that I had written before reading the above part, justifying the suggested change:

> Good catch! It makes sense to pin the npm version. However, we install `nodejs` at `latest`, which means that we are at risk of `apt` packages updates when the latest version of Node becomes incompatible with `npm@9` ([not soon](https://nodejs.org/en/download/releases), but still Node.js v16 is out of LTS at the end of September, so might happen sooner than we think). I thus believe that we should also pin the Node version.

```suggestion
    state: latest # The major version of NodeJS is provided by the NodeSource repository defined in the task above
```